README.md

```diff
@@ -109,6 +109,15 @@ If everything is successful, you should see something like this:
 ```
 (master)$ Successfully rebased and updated refs/heads/master.
 ```
+### Possible issues with merging
+Safe merging strategy:
```

Comment:
from [git-scm](http://git-scm.com/docs/git-merge)
> With --no-commit perform the merge but pretend the merge failed and do not autocommit, to give the user a chance to inspect and further tweak the merge result before committing.

I think the additional opportunity to inspect code can be helpful. the ```no-ff``` flag doesn't really make anything "safer" it just maintains evidence that a feature branch once existed, which in the "branch for each task" style that a lot of teams I have worked, keeping the history consistent is important.